sympy/sets/fancysets.py

```diff
@@ -443,3 +445,451 @@ def _sup(self):
+        >>> a, b = Interval(2, 5), Interval(4, 8)
+        >>> c = Interval(0, 2*S.Pi)
+        >>> c1 = ComplexPlane(a*b)
+        >>> c1.measure
+        12
+        >>> c2 = ComplexPlane(a*c, polar=True)
+        >>> c2.measure
+        6*pi
+
+        """
+        return self.sets._measure
+
+    def _contains(self, other):
+        from sympy.functions import arg, Abs
+
+        # self in rectangular form
+        if not self.polar:
+            re, im = other.as_real_imag()
+            for element in self.psets:
+                if And(element.args[0]._contains(re),
+                        element.args[1]._contains(im)):
+                    return True
+            return False
+
+        # self in polar form
+        elif self.polar:
+            if S(other).is_zero:
+                r, theta = S(0), S(0)
+            else:
+                r, theta = Abs(other), arg(other)
+            for element in self.psets:
+                if And(element.args[0]._contains(r),
+                        element.args[1]._contains(theta)):
+                    return True
+                return False
+
+    def _intersect(self, other):
+
+        if other.is_ComplexPlane:
+            # self in rectangular form
+            if (not self.polar) and (not other.polar):
+                return ComplexPlane(Intersection(self.sets, other.sets))
+
+            # self in polar form
+            elif self.polar and other.polar:
+                r1, theta1 = self.a_interval, self.b_interval
+                r2, theta2 = other.a_interval, other.b_interval
+                new_r_interval = Intersection(r1, r2)
+                new_theta_interval = Intersection(theta1, theta2)
+
+                # 0 and 2*Pi means the same
+                if ((2*S.Pi in theta1 and S(0) in theta2) or
+                   (2*S.Pi in theta2 and S(0) in theta1)):
+                    new_theta_interval = Union(new_theta_interval,
+                                               FiniteSet(0))
+                return ComplexPlane(new_r_interval*new_theta_interval,
+                                    polar=True)
+
+        if other is S.Reals:
+            return other
+
+        if other.is_subset(S.Reals):
+            new_interval = []
+
+            # self in rectangular form
+            if not self.polar:
+                for element in self.psets:
+                    if S.Zero in element.args[0]:
+                        new_interval.append(element.args[0])
+                new_interval = Union(*new_interval)
+                return Intersection(new_interval, other)
+
+            # self in polar form
+            elif self.polar:
+                for element in self.psets:
+                    if (0 in element.args[1]) or (S.Pi in element.args[1]):
+                        new_interval.append(element.args[0])
+                new_interval = Union(*new_interval)
+                return Intersection(new_interval, other)
+            return None
```

Comment:
This and [this](https://github.com/sympy/sympy/pull/9463/files#diff-c1c17bdd28aafd400bf9415c7d048382R888) is also not covered. Tests for these should check if we are getting an unevaluated `Union` or `Intersection` object if the evaluation fails.